src/components/apiDelegation/reusables/OverviewPageContent/OverviewPageContent.tsx

```diff
@@ -152,9 +152,14 @@ export const OverviewPageContent = ({
             </Button>
           </div>
         )}
-        <Panel title={t('api_delegation.card_title')}>
-          {t('api_delegation.api_panel_content')}
-        </Panel>
+        <div className={classes.infoPanelWrapper}>
+          <Panel
+            title={t('api_delegation.card_title')}
+            forceMobileLayout
+          >
+            {t('api_delegation.api_panel_content')}
+          </Panel>
+        </div>
         <div className={classes.pageContentContainer}>
           <h2 className={classes.apiSubheading}>{accessesHeader}</h2>
           <div className={classes.editButton}>
```

Comment:
The left margin here looks unnecessary big here. As the designs state, the margin should be aligned with the title of the accordion header.
![image](https://user-images.githubusercontent.com/22012997/216382114-b2868158-a804-4524-8530-25b8f7439bf5.png)

Reply:
Like this, I can't even see the delete buttons, neither in the header or the content... 😅
![image](https://user-images.githubusercontent.com/22012997/216382389-8047ac52-8f24-4e4f-b9be-dd2947e4dc6c.png)